pkg/webhook/risingwave_validating_webhook.go

```diff
@@ -74,6 +74,14 @@ func (v *RisingWaveValidatingWebhook) validateGlobal(path *field.Path, global *r
 		}
 	}
 
+	if global.Resources.Limits.Cpu().Cmp(*global.Resources.Requests.Cpu()) == -1 {
+		fieldErrs = append(fieldErrs, field.Required(path.Child("resources", "cpu"), "insufficient cpu resource"))
+	}
+
+	if global.Resources.Limits.Memory().Cmp(*global.Resources.Requests.Memory()) == -1 {
+		fieldErrs = append(fieldErrs, field.Required(path.Child("resources", "memory"), "insufficient memory resource"))
+	}
+
```

Comment:
We should also validate the resources in the group template, if the template is not nil, at line 63 above. Since the `validateGlobal` also invokes the `validateGroupTemplate`, I think we can move the checks there.